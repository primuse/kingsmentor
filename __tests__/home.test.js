// import { shallow } from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Home from '../src/components/home.jsx';

// const wrapper = shallow(<Home />);

describe('Home Page component', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <MemoryRouter>
            <Home />
        </MemoryRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
