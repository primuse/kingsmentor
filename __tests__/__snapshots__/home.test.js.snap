// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page component renders correctly 1`] = `
Array [
  <div
    className="container banner"
  >
    <div>
      <img
        className="sc-htpNat fRSOem"
        src="https://yt3.ggpht.com/a-/AN66SAy46JU-97ZuX23xSTCL0TH4oOsnIGmyf4qUTg=s900-mo-c-c0xffffffff-rj-k-no"
        width="60px"
      />
    </div>
    <h2>
      Making things work ... and work well - Mobile Engineer
    </h2>
    <div
      className="nav"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        <h5>
          GITHUB
        </h5>
      </a>
      <div
        className="icons"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          <img
            className="sc-htpNat gDtXvd"
            src="test-file-stub"
            width="20px"
          />
        </a>
        <a
          href="/"
          onClick={[Function]}
        >
          <img
            className="sc-htpNat gDtXvd"
            src="test-file-stub"
            width="20px"
          />
        </a>
      </div>
    </div>
  </div>,
  <div
    className="post-feed container"
  >
    <div
      className="card"
    >
      <div
        className="card-content"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          <header>
            Preventing Unwanted Content from Google Keyboard
          </header>
          <p>
            This is especially useful if your project includes libraries written in native code (.so files) which support different CPU architectures. You can split your APK per architecture.
          </p>
        </a>
        <footer>
          <a
            className="avatar-cont sc-bdVaJa jvfdnP"
            href="/"
          >
            <div
              className="tooltip sc-bwzfXH bhESHo"
            >
              Nosa Belvi
            </div>
            <img
              className="sc-htpNat XITWc"
              src="test-file-stub"
            />
          </a>
          <h5>
            2 MIN READ
          </h5>
        </footer>
      </div>
    </div>
    <div
      className="card"
    >
      <div
        className="card-content"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          <header>
            Preventing Unwanted Content from Google Keyboard
          </header>
          <p>
            This is especially useful if your project includes libraries written in native code (.so files) which support different CPU architectures. You can split your APK per architecture.
          </p>
        </a>
        <footer>
          <a
            className="avatar-cont sc-bdVaJa jvfdnP"
            href="/"
          >
            <div
              className="tooltip sc-bwzfXH bhESHo"
            >
              Nosa Belvi
            </div>
            <img
              className="sc-htpNat XITWc"
              src="test-file-stub"
            />
          </a>
          <h5>
            2 MIN READ
          </h5>
        </footer>
      </div>
    </div>
    <div
      className="card"
    >
      <div
        className="card-content"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          <header>
            Preventing Unwanted Content from Google Keyboard
          </header>
          <p>
            This is especially useful if your project includes libraries written in native code (.so files) which support different CPU architectures. You can split your APK per architecture.
          </p>
        </a>
        <footer>
          <a
            className="avatar-cont sc-bdVaJa jvfdnP"
            href="/"
          >
            <div
              className="tooltip sc-bwzfXH bhESHo"
            >
              Nosa Belvi
            </div>
            <img
              className="sc-htpNat XITWc"
              src="test-file-stub"
            />
          </a>
          <h5>
            2 MIN READ
          </h5>
        </footer>
      </div>
    </div>
    <div
      className="card"
    >
      <div
        className="card-content"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          <header>
            Preventing Unwanted Content from Google Keyboard
          </header>
          <p>
            This is especially useful if your project includes libraries written in native code (.so files) which support different CPU architectures. You can split your APK per architecture.
          </p>
        </a>
        <footer>
          <a
            className="avatar-cont sc-bdVaJa jvfdnP"
            href="/"
          >
            <div
              className="tooltip sc-bwzfXH bhESHo"
            >
              Nosa Belvi
            </div>
            <img
              className="sc-htpNat XITWc"
              src="test-file-stub"
            />
          </a>
          <h5>
            2 MIN READ
          </h5>
        </footer>
      </div>
    </div>
    <div
      className="card"
    >
      <div
        className="card-content"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          <header>
            Preventing Unwanted Content from Google Keyboard
          </header>
          <p>
            This is especially useful if your project includes libraries written in native code (.so files) which support different CPU architectures. You can split your APK per architecture.
          </p>
        </a>
        <footer>
          <a
            className="avatar-cont sc-bdVaJa jvfdnP"
            href="/"
          >
            <div
              className="tooltip sc-bwzfXH bhESHo"
            >
              Nosa Belvi
            </div>
            <img
              className="sc-htpNat XITWc"
              src="test-file-stub"
            />
          </a>
          <h5>
            2 MIN READ
          </h5>
        </footer>
      </div>
    </div>
    <div
      className="card"
    >
      <div
        className="card-content"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          <header>
            Preventing Unwanted Content from Google Keyboard
          </header>
          <p>
            This is especially useful if your project includes libraries written in native code (.so files) which support different CPU architectures. You can split your APK per architecture.
          </p>
        </a>
        <footer>
          <a
            className="avatar-cont sc-bdVaJa jvfdnP"
            href="/"
          >
            <div
              className="tooltip sc-bwzfXH bhESHo"
            >
              Nosa Belvi
            </div>
            <img
              className="sc-htpNat XITWc"
              src="test-file-stub"
            />
          </a>
          <h5>
            2 MIN READ
          </h5>
        </footer>
      </div>
    </div>
  </div>,
  <div
    className="footer"
  >
    <div
      className="container"
    >
      <h4>
        KingsmentorÂ© 2019
      </h4>
      <nav>
        <a
          href="/"
          onClick={[Function]}
        >
          Latest Posts
        </a>
        <a
          href="/"
          onClick={[Function]}
        >
          Twitter
        </a>
      </nav>
    </div>
  </div>,
]
`;
